meta/meta.go

```diff
@@ -76,6 +77,8 @@ var (
 	mPolicyPrefix     = "Policy"
 	mPolicyGlobalID   = []byte("PolicyGlobalID")
 	mPolicyMagicByte  = CurrentMagicByteVer
+	mInitDDLTable     = []byte("initDDLTable")
```

Comment:
Rename this variable to `mDDLTableVersion` or any other name？There may be other versions later